pkg/db/db.go

```diff
@@ -28,7 +29,7 @@ type DB struct {
 	BatchSize int
 }
 
-func New(dsn string) (*DB, error) {
+func New(dsn string, timeNow time.Time) (*DB, error) {
```

Comment:
If you saved the "timeNow" on the actual DB struct, would that eliminate a whole lot of argument passing? I think by definition pretty much anything that will use it already has a *db.DB available.

Reply:
If we decide to instead go with querying this value from the db last prow job run timestamp instead of a CLI arg, that could become a function on DB, or perhaps just a standalone func over in query package the that takes a DB.